e2e/cypress/integration/api/iam/02_policies_api.spec.ts

```diff
@@ -1,9 +1,8 @@
-import { describeIfIAMV2, describeIfIAMV2p1 } from '../../constants';
+import { describeIfIAMV2, describeIfIAMV2p1, adminToken } from '../../constants';
 
 describeIfIAMV2('policies API', () => {
-  let adminToken = '';
   const defaultAdminReq = {
-    auth: { bearer: adminToken },
+    headers: { 'api-token': adminToken },
```

Comment:
most of the changes here are swapping the admin user's bearer token for the admin API token saved as a constant.